spec/support/spec_test_helper.rb

```diff
@@ -6,6 +6,40 @@ def login_as(user)
   def login
     login_as(create(:user))
   end
+
+  RSpec::Matchers.define :match_hash do |expected|
```

Comment:
Maybe we should name this `match_presented_hash` or something, since it does more complex transformations?

Reply:
idr exactly how much match_array did when I looked at it. I could use that if you think it's important?